src/pages/SignupLogin/Login.js

```diff
@@ -0,0 +1,110 @@
+/* - Main import(s) 				-- */
+import React, { Component } from 'react';
+
+/* - Style import(s) 			-- */
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './styles/Login.css';
+import SignupLoginHeader from './SignupLoginHeader';
+
+/*/
+*  Component: Login
+*  @props: TBD
+*  @EventHandler(s): TBD
+*  @Description: Login form
+/*/
+class Login extends Component {
+	constructor(props) {
+		super(props);
+		this.handleLoginSubmission = this.handleLoginSubmission.bind(this);
+		this.showSignup = this.showSignup.bind(this);
+		this.handleInputChange = this.handleInputChange.bind(this);
+
+		this.state = {
+			email: '',
+			password: '',
+			rememberMe: false,
+		};
+	}
+
+	/* Change email, password, checkbox values when user interacts with them */
+	handleInputChange(event) {
+		this.setState({
+			[event.target.name]: event.target.type === 'checkbox' ? event.target.checked : event.target.value,
+		});
+	}
+
+	/* Propagate data to parent component to send to backend */
+	handleLoginSubmission(event) {
+		this.props.onReceiveLoginInformation(this.state.email, this.state.password, this.state.rememberMe);
+		event.preventDefault();
```

Comment:
should have `event.preventDefault()` function before you call any other function

Reply:
Why is this?

Reply:
to prevent data loss. It's highly unlikely that it would ever happen but hypothetically the form could submit before calling the preventDefault() function which refreshes the DOM and could potentially lose the data in the state of that component. Again, it's highly unlikely it would happen, more of a precaution